### DISCUSSION TITLE
Hardware and different liveries

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#6559] [General]
Hi developer,

I made some strange observation and I can´t figure out why the follwing things happen:

I fly always the last developer version and I am using some hardware. A FCU, EFIS and a MCDU. But I can´t use this hardware for all liveries I downloaded?! There are some liveries where every part of the hardware works 100 % and there are liveries where the link software (FCU) closes himself after a few seconds and the MCDU hasn´t all functions. As well the EFIS. Why??

Any suggestions what the point of difference is?

Greets Klaus

**Comment by tracernz** [comment]
Make sure the liveries don't have a `SimObjects/AirPlanes/FlyByWire_A320_NEO/panel/panel.cfg`. If they do you can delete.

**Reply by CptBayer** [reply]
Hi tracernz,

thanx for your hint. But looks like the reason must be another, cause no panel folder inside!? I have a simple solution for me, I copy only the textures in the texture folder of a 100 % working livery .... no problems till now ...

greets from germany
